pkg/kubelet/kubeletconfig/defaults.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package config
```

Comment:
admitting, I am not good with names. Why not keep it just `config`?

Reply:
Do you mean why "kubeletconfig" rather than "config"? Mostly because I don't want people getting confused between the old config package, and kubelet config. And also because `kubeletconfig` already exists, and didn't seem like it needed changing. I'm definitely open to renaming that one though.